### PR TITLE
UNI-501 Identify first csv row with numbers and no date as header row.

### PR DESCRIPTION
**Pull request by vitaly-krugl**

### CODE REVIEW

**Comment by vitaly-krugl** [comment]
@marionleborgne: Ready for review

File: unicorn/tests/js/fixtures/numeric-only-header.csv [hunk 1]
```diff
@@ -0,0 +1,401 @@
+0,1,2,3,4
```
Comment (marionleborgne) [inline unicorn/tests/js/fixtures/numeric-only-header.csv]: what happens if `0` is a unix timestamp? there is now way to disambiguate a data row from a header row.

Comment (vitaly-krugl) [inline unicorn/tests/js/fixtures/numeric-only-header.csv]: @marionleborgne, this PR doesn't support unix timestamps. Unix timestamps will be supported in a later PR that will address UNI-435.

Comment (vitaly-krugl) [inline unicorn/tests/js/fixtures/numeric-only-header.csv]: @marionleborgne, also in "there is now way to disambiguate a data row from a header row", did you mean to say "now" or "no"?

Comment (marionleborgne) [inline unicorn/tests/js/fixtures/numeric-only-header.csv]: 👍

**Pull request merged**